tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -301,7 +596,7 @@ public enum ExplorationTestUseCase
 
 public enum ExplorationTestName
 {
-    eShopOnWeb, protobuf, cake, swashbuckle, paket, RestSharp, serilog, polly, automapper, /*ilspy*/
```

Comment:
IIRC, ilspy was commented out, because it was very flaky. Do we have any reason to think that won't be the case now?

Reply:
Fixed in 6853dece4bb3cc91a684157e03ae7d3a18a3de0a